packages/expo-dev-launcher/ios/ReactDelegateHandler/ExpoDevLauncherReactDelegateHandler.swift

```diff
@@ -43,15 +43,14 @@ public class ExpoDevLauncherReactDelegateHandler: ExpoReactDelegateHandler, RCTB
     ExpoDevMenuReactDelegateHandler.enableAutoSetup = false
 
     self.reactDelegate = reactDelegate
-    self.bridgeDelegate = EXRCTBridgeDelegateInterceptor(bridgeDelegate: bridgeDelegate, interceptor: self)
```

Comment:
without the `EXRCTBridgeDelegateInterceptor`, does the current solution support user's custom delegate in AppDelegate, e.g. if there are custom `extraModulesForBridge:`?

Reply:
Good question, let me double check if that still works as expected

Reply:
Hi @Kudo, you were right about this, the current implementation is not working as expected, do you think it would be ok to pass `interceptor` as `ExpoDevLauncherBridgeDelegateHandler`?